### DISCUSSION TITLE
Tokens

### DISCUSSION THREAD

**Discussion by KST99** [community/community#30769] [Code Security]
How can I create a token on GitHub

**Comment by ghost** [comment]
Go to your personal settings, Developer Settings. There you can create tokens and authorize your apps.
[https://github.com/settings/tokens](https://github.com/settings/tokens)

There is a [full manual](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) to answer this question in the docs.
